pkg/core/localize/locales/en/cmd/serviceaccount_create.en.toml

```
@@ -14,6 +14,7 @@ You must specify an output format into which the credentials will be stored.
 - env (default): Store credentials in an env file as environment variables
 - json: Store credentials in a JSON file
 - properties: Store credentials in a properties file, which is typically used in Java-related technologies
+- credentials.properties: Store credentials in a properties file suitable for the Java Kafka client
```

Since we are elaborating the flags here, it should be `java-kafka-properties`
```suggestion
- java-kafka-properties: Store credentials in a properties file suitable for the Java Kafka client
```